pkg/diff/chunk.go

```diff
@@ -62,14 +62,14 @@ func newChunkRange(begin, end interface{}, containBegin, containEnd, noBegin, no
 		noEnd:        noEnd,
 	}
 }
-func getChunksForTable(db DBConfig, table *TableConfig, column *model.ColumnInfo, chunkSize, sample int) ([]chunkRange, error) {
+func getChunksForTable(table *TableInstance, column *model.ColumnInfo, chunkSize, sample int, limits string) ([]chunkRange, error) {
```

Comment:
unify name, s/limits/range

Reply:
range is a keyword in golang, so I use range

Reply:
so, is it possible to put `limits` into `table`

Reply:
table is a  instance for table, seems not suitable